lib/element.ts

```diff
@@ -1061,7 +1061,7 @@ export class ElementFinder extends WebdriverWebElement {
  * @returns {ElementFinder} which identifies the located
  *     {@link webdriver.WebElement}
  */
-export let build$ = (element: ElementHelper, by: ProtractorBy) => {
+export let build$ = (element: ElementHelper, by: typeof By) => {
```

Comment:
The thing is this function is actually called with WebDriver's `By`, not with Protractor's one. See `browser.ts`:
```ts
this.$ = build$(this.element, By);
this.$$ = build$$(this.element, By);
```
Hence this change.